qml/plugins/RestartBluetooth.qml

```diff
@@ -0,0 +1,17 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Utilities 1.0
+
+ActionItem {
+    //% "Bluetooth"
+    title: qsTrId("settings_bluetooth-bluetooth")
```

Comment:
```suggestion
    title: qsTrId("sailfish-tools-he-bluetooth")
```
My understanding is that the "he", "bt" etc. in these tags refer to "header", "button", etc. so the strings you're using probably need a bit of tweaking.